deploy/rbac-permissions-operator-config/03-dedicated-admins-cluster.ClusterRole.yaml

```diff
@@ -20,17 +20,37 @@ aggregationRule:
       operator: In
       values:
       - OperatorGroup
+    # exclude specific operator groups by name (aka owner)
+    - key: olm.owner
+      operator: NotIn
+      values:
+      # platform operator groups
+      - openshift-cluster-monitoring
+      - olm-operators
+      # OSD operator groups
+      - cloud-ingress-operator
+      - rbac-permissions-operator
+      - splunk-forwarder-operator-og
```

Comment:
Shouldn't `openshift-velero` be listed here?

Reply:
It's not deployed via OLM yet, is deployed by MCC.
https://github.com/openshift/managed-cluster-config/tree/master/deploy/managed-velero-operator

Reply:
The exclusion of `openshift-velero` namespace will catch it in the future until we get an operatorgroup.

Reply:
:+1: